lib/fail.js

```diff
@@ -3,6 +3,7 @@ var match = require('pattern-match');
 function ValidationError(message, stack) {
     Error.call(this, message);
     this.stack = stack;
+    this.message = message;
```

Comment:
Is this line necessary? From my reading of the documentation, Error.prototype.message has the message.